src/main/resources/db/migration/V5.10.1.3__8605-support-archival-status.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE datasetversion SET archivalCopyLocation = CONCAT('{"status":"success", "message":"', archivalCopyLocation,'"}') where archivalCopyLocation is not null and not archivalCopyLocation='Attempted';
+UPDATE datasetversion SET archivalCopyLocation = CONCAT('{"status":"failure", "message":"Attempted"}') where archivalCopyLocation='Attempted';
```

Comment:
@qqmyers can you please bump the version number on this Flyway script? I can't deploy:

Caused by: org.flywaydb.core.api.FlywayException: Found more than one migration with version 5.10.1.3
Offenders:
-> /usr/local/payara5/glassfish/domains/domain1/applications/dataverse-5.11/WEB-INF/classes/db/migration/V5.10.1.3__8599-legacy-templ
ates.sql (SQL)
-> /usr/local/payara5/glassfish/domains/domain1/applications/dataverse-5.11/WEB-INF/classes/db/migration/V5.10.1.3__8605-support-arch
ival-status.sql (SQL)

Reply:
Oh, actually, I think we simply need to delete V5.10.1.3__8605-support-archival-status.sql since it's the same as V5.11.0.1__8605-support-archival-status.sql. Does that sound right?

Reply:
This file has been removed.